src/utils/printing.jl

```diff
@@ -126,6 +130,27 @@ function Base.show(io::IO, ::MIME"text/plain", results::Results)
     println(io, "Stage()")
  end
 
+function Base.show(io::IO, period::Dates.Millisecond)
```

Comment:
I think that this should account for the size of the number and not just divisibility.  For example, I think that 45.2 seconds should be reported in seconds rather than milliseconds.